OpenRA.Mods.Common/Traits/BotModules/BotModuleLogic/BaseBuilderQueueManager.cs

```diff
@@ -370,11 +372,38 @@ ActorInfo ChooseBuildingToBuild(ProductionQueue queue)
 			return null;
 		}
 
+		bool AdjacentBuildingAt(CPos cell)
+		{
+			foreach (var adjacent in CVec.Directions)
+			{
+				var adjacentCell = cell + adjacent;
+				if (buildingInfluence.AnyBuildingAt(adjacentCell))
+					return true;
```

Comment:
This should probably just include own buildings but not enemy ones, shouldn't it? We also might want to exclude things walls if it isn't already here.

Reply:
I think the type of building and the owner does not matter. The AI should still not try to block its units from moving through a base as that can lead to impossible paths that are horrible for performance.

Reply:
Fair enough.